app/pb/api.proto

```diff
@@ -314,6 +334,7 @@ message UpdateProfileReq {
   NullableBoolValue parking = 119;
   ParkingDetails parking_details = 120;
   NullableBoolValue camping_ok = 121;
+  RepeatedLanguageAbilityValue language_abilities = 25;
```

Comment:
If we limit the language set from the full ISO 639-3 set need to state the valid languages here in the API description.